lib/phlex/component.rb

```diff
@@ -70,7 +70,7 @@ def doctype
     end
 
     def raw(content)
-      @_target << content
+      @_target << content.to_s
```

Comment:
I don’t think we should implicitly convert raw content to a string. Raw output should be really explicit.